shell/src/main/java/tachyon/shell/command/CountCommand.java

```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "count <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the number of files and directories matching the specified prefix.";
```

Comment:
the class description has this problem too. What do other reviewers think? @aaudiber